lib/watir/elements/image.rb

```diff
@@ -16,18 +16,6 @@ def loaded?
       )
     end
 
-    #
-    # Returns the image's width in pixels.
-    #
-    # @return [Integer] width
-    #
-
-    def width
```

Comment:
Hmmm, I would have sworn I had an issue and this was the fix. Maybe I screwed up something else during the testing. I'll give it a try in elements and report back.